cocos/core/math/mat4.ts

```diff
@@ -267,7 +266,8 @@ export class Mat4 extends ValueType {
         const a30 = a.m12; const a31 = a.m13; const a32 = a.m14; const a33 = a.m15;
 
         // Cache only the current line of the second matrix
-        let b0 = b.m00, b1 = b.m01, b2 = b.m02, b3 = b.m03;
+        let b0 = b.m00; let b1 = b.m01; let b2 = b.m02; let
+            b3 = b.m03;
```

Comment:
Same as previous, but the line length doesn't seem overflow, why it's happening here